### ISSUE TITLE
[Rogue]  Scaling of Deadly poison + Eviscerate + Envenom 

### ISSUE THREAD

**Issue by zarkovod** [dalaranwow/dalaran-wow#5360]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)

**Expected behaviour**: Tell us what should happen instead

**Steps to reproduce the problem**:

1. Deadly poison and Instant poison are not scaling properly with Attack power as they should ( curently they are doing like 50% lower dmg )
2. Eviscerate dmg is not scaling properly ( it pulls crits for 20k dmg, which is too low ) the scaling from Attack power is 50% reduced.
3. Envenom dmg is not scaling properly , it is doing too low dmg aswell as Eviscerate.
Talent Hack and slash is not working properly ( nearly 1% instead 5% ).

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Rushor** [comment]
hack and slash will be handled here https://github.com/dalaranwow/dalaran-wow/issues/5242

**Comment by robinsch** [comment]
### Here are the scaling we use:
Eviscerate: damage + attackPower * combo * 0.07
Envenom: damage * doses + attackPower * 0.09 * combo
Deadly Poison: damage per tick + attackPower * 0.03 (per tick - so it's 12 / 4)
Instant Poison: damage + attackPower * 0.1

Those scaling seems to be correct. Closing this until we get more information from you.

**Comment by AttendantTock** [comment]
Hello,

To provide data, I did some runs on the PTR for Eviscerate.

Eviscerate formula (source Rawr, not showing talent/damage modifiers to keep it simple):
total = baseDamage + comboDamage * comboNumber
with:
- baseDamage = 254
- comboDamage = 370 + AP * 0.07

Test runs versus a naked 80 toon, no talents or modifiers, about 20 eviscerates each:
- 571 AP, 1 combo: damage expected 664, damage measured 699
- 571 AP, 5 combo: damage expected 2303, damage measured 2402
- 1176 AP, 1 combo: damage expected 706, damage measured 755
- 1176 AP, 5 combo: damage expected 2516, damage measured 2597
- 1786 AP, 1 combo: damage expected 749, damage measured 775
- 1786 AP, 5 combo: damage expected 2729, damage measured 2837

If anything, the damage is slightly superior to what is expected, but nowhere near a 50% attack power scaling nerf.

**Comment by Bajeru** [comment]
fucking naxx geared rogues from other server outdps icc geared rogues here. when is it noticeable that everywhere else than this server rogues work

**Comment by HiNico** [comment]
The problem with low damage form the ability Eviscerate and Deadly poison is in regards to Prey on the Weak issues and not the abilities themself right ? 
Do not think rogues are gimped because of scaleing, but buggy talents.

**Comment by InshadowsAlgalon** [comment]
Tested Pray of The Weak, on the player (not mob) - works fine. The only aspect not tested is having full hp, and shield on urself, while mob is below 100% hp.

**Comment by InshadowsAlgalon** [comment]
will update
